ddtrace/bootstrap/sitecustomize.py

```diff
@@ -78,6 +78,7 @@ def drop(module_name):
             "attr",
             "google",
             "google.protobuf",  # the upb backend in >= 4.21 does not like being unloaded
+            "wrapt",
```

Comment:
Just for my own sanity check...

We added `wrapt` here to make sure we keep the implicitly imported `wrapt` module from `opentelemetry-sdk`. If we don't add this here, we would remove it from `sys.modules` during unloading and it wouldn't be able to be used via `import wrapt` in the other files.

So if `wrapt` ends up _not_ getting installed by `opentelemetry-sdk` in the future, do we need to do anything? Or is that handled by having it in the `project.toml` file as a dependency?

Reply:
The unloading is so we can keep our own copy of modules... however, in a _ton_ of places we do things like `if isinstance(thing, wrapt.FunctionWrapper)`, but if we have our own copy, some of that stuff may start failing. Instead, I think we definitely 100% want to use the same `wrapt` module for the whole process.

I didn't notice any issues, this is my hunch/instinct based on our usage.